experimental/StandardFiniteFields/test/runtests.jl

```diff
@@ -0,0 +1,44 @@
+using Test
+using Oscar
+GAP.Packages.load("StandardFF")
+
```

Comment:
```suggestion

@testset "standard_finite_field unit tests" begin
```
I would recommend starting your testset already here. This has the upside of not cluttering the global namespace with your functions `MyPoly` etc., sinse `@testset` introduces a new scope.